builtin/providers/vsphere/README.md

```diff
@@ -54,4 +54,59 @@ Click on "+" icon (Create role action), give it appropraite name and select foll
 These settings were tested with [vSphere 6.0](https://pubs.vmware.com/vsphere-60/index.jsp?topic=%2Fcom.vmware.vsphere.security.doc%2FGUID-18071E9A-EED1-4968-8D51-E0B4F526FDA3.html) and [vSphere 5.5](https://pubs.vmware.com/vsphere-55/index.jsp?topic=%2Fcom.vmware.vsphere.security.doc%2FGUID-18071E9A-EED1-4968-8D51-E0B4F526FDA3.html). For additional information on roles and permissions, please refer to official VMware documentation.
 
 This section is a work in progress and additional contributions are more than welcome.
- 
+
+# Acceptance Tests
+
+The VMware vSphere provider's acceptance tests require the above provider
+configuration fields to be set using the documented environment variables.
+
+## Environment Variables
+
+In addition, the following environment variables are used in tests, and must be
+set to valid values for your VMware vSphere environment:
+
+ * VSPHERE\_IPV4\_GATEWAY
+ * VSPHERE\_IPV4\_ADDRESS
+ * VSPHERE\_IPV6\_GATEWAY
+ * VSPHERE\_IPV6\_ADDRESS
+ * VSPHERE\_NETWORK\_LABEL
+ * VSPHERE\_NETWORK\_LABEL\_DHCP
+ * VSPHERE\_TEMPLATE
+
+The following environment variables depend on your vSphere environment:
+
+ * VSPHERE\_DATACENTER
+ * VSPHERE\_CLUSTER
+ * VSPHERE\_RESOURCE\_POOL
+ * VSPHERE\_DATASTORE
+
+The following additional environment variables are needed for running the
+"Mount ISO as CDROM media" acceptance tests.
+
+ * VSPHERE\_CDROM\_DATASTORE
+ * VSPHERE\_CDROM\_PATH
+
+ The following environment variable is needed from running the "Create With Existing VMDK"
+ acceptance tests
+
+ * VSPHERE\_VMDK\_PATH
+
+ Refer to the TF documentation in regards to logging but at the current time
+
+ * TF\_LOG=DEBUG
+
+ As always refer to the acceptance test source code for the latest variables.
+
+## Running tests
+
+Once all these variables are in place, the tests can be run like this:
+
+```
+make testacc TEST=./builtin/providers/vsphere
+```
+
+In order to run a single test `TESTARGS` Envrionment variable can be set:
+
+```
+export TESTARGS="-run TestAccVSphereVirtualMachine_ipv4Andipv6"
```

Comment:
I use `make testacc TEST=./builtin/providers/vsphere TESTARGS='-run=TestAccVSphereVirtualMachine_ipv4Andipv6'`. I think it is more flexible, since you can keep a few versions in shell history